app/helpers/application_helper.rb

```diff
@@ -165,17 +165,14 @@ def active_nav_li(link)
   end
 
   def show_time(length)
-    h = length / 60
-    min = length - h * 60
+    h, min = length.divmod(60)
```

Comment:
+1

``
nil.divmod(60)
``

will throw an undefined method  exception!

(I know that it did it before, however, I think it would be nice to fix it).